pkg/manager/member/tikv_group_member_manager.go

```diff
@@ -645,6 +645,20 @@ func getNewTiKVSetForTiKVGroup(tg *v1alpha1.TiKVGroup, tc *v1alpha1.TidbCluster,
 		})
 	}
 	tikvContainer.Env = util.AppendEnv(env, baseTiKVSpec.Env())
+	var storeLabels map[string]string
+	if tg.Spec.InitializeStoreLabels != nil {
+		storeLabels = tg.Spec.InitializeStoreLabels
+	} else {
+		storeLabels = map[string]string{}
+	}
+	storeLabels["groupName"] = tg.Name
```

Comment:
For each TiKVGroup instance, we will set `groupName`: <tikvgroup-name> as its store label. @rleungx